src/main/java/com/jungdam/diary/domain/vo/Bookmark.java

```diff
@@ -7,13 +7,16 @@
 public class Bookmark {
 
     @Column(name = "diary_bookmark")
-    private Boolean bookmark;
-
-    protected Bookmark() {
+    private Boolean bookmark = false;
 
+    public Bookmark() {
     }
 
-    public Bookmark(Boolean bookmark) {
-        this.bookmark = bookmark;
+    public void check() {
+        if (this.bookmark) {
+            this.bookmark = false;
+            return;
+        }
+        this.bookmark = true;
```

Comment:
```suggestion
    public void check() {
        this.bookmark = !this.bookmark;
```
무엇을 위한 check인지는 모르겠는데 좀 더 간결하게 표현 가능할 것 같아요.
true일 경우 false로 변경하고, false일 경우 true를 반환하는 로직이 맞다면..?

아 클릭했을 때 북마크/북마크 취소인가요?

Reply:
넵넵 북마크에 대한 기능인데 아직 사용하지 않는 걸 올렸네요 ㅜㅜㅜ
해당 로직은 지웠다가 추후에 다시 추가할게용~